scipy/interpolate/interpnd.pyx

```diff
@@ -73,36 +73,12 @@ class NDInterpolatorBase:
         else:
             self.tri = None
 
+        self.ndim = ndim
```

Comment:
A new attribute, a leftover from previous iterations?

Reply:
Yes, removing it.

Reply:
Yes, removed.